### ISSUE TITLE
save the visualization

### ISSUE THREAD

**Issue by hard-workingXu** [hailanyi/VirConv#69]
Hello, thank you very much for your excellent work, successfully trained and tested the model using the code on github, but wondering how you can visualize the results of the predictions and save them?

**Comment by ck-png** [comment]
Did you solve it?

**Comment by hard-workingXu** [comment]
No, I tried to use the visualization method in PCDET, but it doesn't look like the 3D box is aligned

**Comment by chenshans** [comment]
I wonder if you've solved it. Would you mind sharing the solution?

**Comment by zhaojinbiao** [comment]
> No, I tried to use the visualization method in PCDET, but it doesn't look like the 3D box is aligned
me too,  the 3D box is not aligned
